lib/pure/base64.nim

```diff
@@ -144,40 +146,23 @@ template encodeImpl() {.dirty.} =
 proc encode*[T: SomeInteger|char](s: openArray[T], safe = false): string =
```

Comment:
So ... you left this one as `SomeInteger`? Bad idea.

Reply:
> > @Araq, What about code like encode @[111, 113, 32] ? should it be deprecated ?
> 
> I think so but there is no RFC for it so better leave it as it is.

So code like that just breaks ?

Reply:
Wait, so can you explain why this procedure was left as-is?

Reply:
> Wait, so can you explain why this procedure was left as-is?

> > @Araq, What about code like encode @[111, 113, 32] ? should it be deprecated ?
> 
> 
> I think so but there is no RFC for it so better leave it as it is.
> 

" leave it as it is"

Reply:
(2)